### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by dombarrieau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,15 +79,29 @@ player.
...
-  * Plan-domizer uses the spoiler log JSON format
+  * Spoiler log is now in JSON format.
+  * Plan-domizer uses the spoiler log JSON format.
 * Cow Sanity
   * Playing Epona's Song for a cow for the first time gives an item.
 * Starting Age Option
   * Can now start as child, adult, or random.
+* Entrance Randomizer
+  * Ability to randomize dungeon entrances or all interior entrances.
+  * Gossip stone added to Lake Hylia to change water level after beating Morpha to keep dungeon entrance accessible with Gold Scales.
```
Comment (Roman971) [inline README.md]: It seems everyone is still confused about this, the new gossip stone in Lake Hylia which allows you to freely change the water level after beating Morpha is not related to Entrance Shuffle whatsoever. It's there in all settings and fully part of the randomizer now.
Also it's probably a good idea to mention the "Child May Drain Lake Hylia" option in the changelog, which is also not directly related to Entrance Shuffle btw.

Comment (dombarrieau) [reply README.md]: Will update, thanks for the info :)

Comment (Roman971) [reply README.md]: No worries, the rest seems good to me tho. 👍

Comment (dombarrieau) [reply README.md]: Always useful getting extra pairs of eyes on these changes <3

**Comment by TestRunnerSRL** [comment]
```
* Desert Colossus Hands are not logically part of Spirit Temple
```
should say `now` not `not`.

```
* Progressive items can no longer show up as foolish choice when they aren't.
* Vanilla shop items have correct price in spoiler log with shopsanity enabled.
```
I'm not sure either of these are fixed from 4.0, and are interim.

**Comment by dombarrieau** [comment]
Yup, that's a typo :P
The progressive item one I can remove, but the shop one someone brought up that it happened a in 4.0, and Bee confirmed it is fixed in dev, so I figured it was safe to add to the changelog. If you'd rather it remove I can do that

**Pull request merged**